wallet/androidTest/testNet3/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is only needed to generate the compiled resource file `R.txt` for Espresso tests, without it, test builds fail.